modules/auxiliary/admin/mysql/mysql_enum.rb

```diff
@@ -118,12 +124,15 @@ def run
       password_field = 'password'
     end
 
+    # rubocop:disable Style/ZeroLengthPredicate
+    # MySQL query response does not support `.empty?`
```

Comment:
I've create a PR to address this:

https://gitlab.com/tmtms/ruby-mysql/-/merge_requests/12

No ETA on merge. This should not be considered a blocker for this PR.